base/base_tests/newtype_test.cpp

```diff
@@ -96,4 +96,19 @@ UNIT_TEST(NewTypeMember_Operations)
   a ^= Int(7);
   TEST_EQUAL(a, Int(10 ^ 7), ());
 }
+
+namespace test_output
```

Comment:
I need this, cause I have to different DebugPrints: one from above and one defined by a macro.